Blog_part_1.md

```diff
@@ -0,0 +1,359 @@
+Load Testing with Locust
+========================
+
+We believe it's critical to test our systems under load to attempt to understand the impact of system traffic. Not only the increase traffic but also the different styles of traffic. From bursty where a wall of traffic comes within a few minutes or even seconds to traffic that increases more uniformly. 
+
+Generating load to exercise our system helps us understand how to configure our system. The different characteristics of activity can have a significant impact on how we both design and configure our systems. For example on a system like Google App Engine Google gives us the ability to configure how many idle machines (they call them instances) Google will provision beyond currently active instances. This allows us to have a buffer of machines to take on load if a wall of traffic hits. We can also configure how long a request since sits in the pending queue waiting for an instance to open up for use. In a latency sensitive environment we may configure this number to be quite low, combined with adding a buffer of idle instances when new traffic comes in it spends a short amount of time in the queue as the scheduler quickly sees that there are no available instances but thankfully instead of waiting for the instances to be spun up our traffic can be routed to the idle instances that have already been spun up. The negative of this of course is the cost of our application will go up. We may be over provisioned and now have wasted compute costs.
```

Comment:
Delete "The different characteristics of activity can have a significant impact on how we both design and configure our systems."
s/For example: on a/For example: a

> In a latency sensitive environment we may configure this number to be quite low, combined with adding a buffer of idle instances when new traffic comes in it spends a short amount of time in the queue as the scheduler quickly sees that there are no available instances but thankfully instead of waiting for the instances to be spun up our traffic can be routed to the idle instances that have already been spun up.

Run-on sentence.